### DISCUSSION TITLE
What is server keyword in package.json

### DISCUSSION THREAD

**Discussion by craph** [requarks/wiki#7264] [Help / Questions] [answered]
Hello,

Anyone can help me to understand what is the "server" here in the package.json  : 
```
"scripts": {
    "start": "node server",
```

In NodeJs there is no server option ? 

Am I missing something ?
Thank you very much.

Best regards,

**Comment by fnafox** [comment]
Path reference. Start node in directory "server".

**Reply by craph** [reply]
Hello @fnafox ,

Thank you very much for the explanation.

Is there any official documentation about that for node ?

**Reply by fnafox** [reply]
I am not a node developer but this should be it
https://nodejs.org/api/cli.html#program-entry-point

**Reply by craph** [reply, marked as answer]
Thank you very much
